usim_pytest/test_types/test_pipe.py

```diff
@@ -5,13 +5,6 @@
 from ..utility import via_usim, assertion_mode
 
 
-async def aenumerate(aiterable, start=0):
-    count = start
-    async for item in aiterable:
-        yield count, item
-        count += 1
-
-
```

Comment:
What is the intention to include this in this PR?

Reply:
The ``aenumerate`` is now used in ``test_time.py``, so it was effectively "moved".